<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>495</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>833</width>
    <height>495</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>833</width>
    <height>495</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>车牌识别服务器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>10</y>
      <width>821</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabVideo">
     <attribute name="title">
      <string>视频</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabPlate">
     <attribute name="title">
      <string>车牌识别结果</string>
     </attribute>
     <widget class="QPlainTextEdit" name="txtPlateLog">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>10</y>
        <width>800</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabDb">
     <attribute name="title">
      <string>数据库</string>
     </attribute>
     <widget class="QPlainTextEdit" name="txtDbLog">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>10</y>
        <width>800</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabFtp">
     <attribute name="title">
      <string>FTP</string>
     </attribute>
     <widget class="QPlainTextEdit" name="txtFtpLog">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>10</y>
        <width>800</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Serialize</string>
     </attribute>
     <widget class="QPlainTextEdit" name="txtSerializeLog">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>10</y>
        <width>800</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>833</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
